### PR TITLE
Use PatchInstance from lib-common

### PR DESCRIPTION
**Pull request by gibizer**
Depends-on: https://github.com/openstack-k8s-operators/lib-common/pull/117 (merged)

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -90,3 +90,5 @@ require (
 )
 
 replace github.com/openstack-k8s-operators/nova-operator/api => ./api
+
+replace github.com/openstack-k8s-operators/lib-common/modules/common => github.com/abays/lib-common/modules/common v0.0.0-20221215201557-feb92dd06374
```
Comment (bogdando) [inline go.mod:94]: This is LGTM, although we should not approve it for merge until this replacement goes away and the dependency patch is merged, correct?

Comment (gibizer) [reply go.mod:94]: Yes, we need to land the lib common change first and then remove this replace line

Comment (gibizer) [reply go.mod:94]: I have a PR in flight to catch such replace lines and prevent merging them #213

**Comment by gibizer** [comment]
Updated to the latest lib-common proposal.

**Review by bogdando — COMMENTED** [review]
the dependency has been merged

**Comment by gibizer** [comment]
> the dependency has been merged

yepp I need to get back to this bump the lib-common dep and remove the replace line from the go.mod

**Review by bogdando — APPROVED** [review]

**Pull request merged**